code/grant/html_lab04/templates/index.html

```diff
@@ -0,0 +1,249 @@
+<!DOCTYPE html>
+<html lang="en">
+<!-- HEAD SECTION -->
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/devicons/devicon@v2.15.1/devicon.min.css">
+
+    <link rel="stylesheet" href="{{url_for('static', filename='styles.css')}}">
+    <title>Grant's Portfolio</title>
+</head>
+<!-- HEAD SECTION END -->
+<!-- NAVBAR SECTION -->
+
+<body>
+    <nav class="navbar navbar-expand-lg bg-dark navbar-dark" id=top-bar>
+        <div class="container-fluid">
+            <a href="file:///C:/Users/grant/Documents/class_liger/code/grant/html_lab03/templates/HOME.html"
+                class="navbar-brand">Grant's Portfolio</a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbutton">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+
+            <div class="collapse navbar-collapse" id="navbutton">
+                <ul class="navbar-nav ms-auto">
+                    <li class="nav-item">
+                        <a href="file:///C:/Users/grant/Documents/class_liger/code/grant/html_lab03/templates/ABOUT-US.html"
+                            class="nav-link">ABOUT ME</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="file:///C:/Users/grant/Documents/class_liger/code/grant/html_lab03/templates/HOME.html"
+                            class="nav-link">HOME</a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="file:///C:/Users/grant/Documents/class_liger/code/grant/html_lab03/templates/CONTACT.html"
+                            class="nav-link">CONTACT</a>
+                    </li>
+                </ul>
+
+            </div>
+        </div>
+
+
+
+
+    </nav>
+    <section class="container-fluid bg-dark text-light" id="initials">
+        <div class="d-flex justify-content-center align-items-center ">
+            <p style="font-size:17rem">G</p>
```

Comment:
You should avoid mixing style and raw CSS when using a framework such as bootstrap. There are methods to customize bootstrap if necessary. Sometimes it will work fine, but other times it may not work or will even break things. In this case, it was likely your best option for the giant G but in other cases, bootstrap can override CSS no matter what load order is used. (Most of this is advanced. Short version: Avoid using vanilla CSS with a framework)